### PR TITLE
[WIP] Adding docker-compose functionality

### PR DESCRIPTION
**Pull request by pryorda**
This is missing the agent and its all rough. You can get it up by typing docker-compose up -d elastic && docker-compose up natlas-server

### CODE REVIEW

**Comment by 0xdade** [comment]
Heads up, I know we talked about this PR via DM but something I just noticed. I won't accept the PR into the master branch. Please retarget the PR against the v0.6.x branch, where active development is happening. The master branch is generally reserved for either version release merges or if I have to write/correct some documentation that is important to have correct on master (usually have to do this if I messed up a release).

File: natlas-server/run-server.sh [hunk 1]
```diff
@@ -21,5 +21,5 @@ else
 	echo "$(date) : Production" >> logs/start.log
 	echo "$(date) : Production"
 
-	gunicorn -b 127.0.0.1:5000 natlas-server:app
+	gunicorn -b 0.0.0.0:5000 natlas-server:app
```
Comment (tophertimzen) [inline natlas-server/run-server.sh]: If someone wants to run the server without docker, this is going to expose port 5000 on all ports. Is there a way around this so we don't need to change run-server.sh?

**Comment by tophertimzen** [comment]
I added a comment to the changes made in `run-server.sh`, but did not start a review. I think my comment needs to be addressed before we merge this into v0.6.x.

File: natlas-server/run-server.sh (hunk 1 above)
Comment (0xdade) [reply natlas-server/run-server.sh]: I'm in agreement, I mentioned this in a DM before the PR got opened but it's a good idea to track it in ticket. Dockerizing should have as little impact on running it in a non-docker environment as possible.

Comment (pryorda) [reply natlas-server/run-server.sh]: I can use bash param expansion to handle this. 

${BIND_ADDRESS:-127.0.0.1}

**Comment by pryorda** [comment]
I'm not hoping for a merge anytime soon. There is a lot of work to be done and maybe we can slate this for 0.7.x?

File: natlas-server/run-server.sh (hunk 1 above)
Comment (pryorda) [reply natlas-server/run-server.sh]: Agreed. My first intention of this is just to get a repeatable dev-stack running. Then optimize for "production" and "security."

Comment (pryorda) [reply natlas-server/run-server.sh]: This should be resolved in the latest commit.

**Comment by pryorda** [comment]
https://github.com/natlas/natlas/pull/171/files#r334204358

**Comment by pryorda** [comment]
Can you guys stand up or install docker? 

Run docker-compose build
Run docker-compose up -d elastic
Run docker-compose up -d natlas-server
Run docker-compose up -d natlas-agent

To view logs or test

docker-compose logs --tail 100 -f

**Comment by pryorda** [comment]
This was fully functional on my end. I could scan, get screenshots, etc.

**Comment by pryorda** [comment]
Can you guys test downloading this, install docker or docker desktop, then run ` make run `, then ` make create-account `?

**Comment by 0xdade** [comment]
What is this pull bot? 🤔

**Comment by pryorda** [comment]
Its an app I configured so that all my forks stay with their upstream

**Comment by pryorda** [comment]
I didn't know it would close this :/

**Pull request closed without merging**